src/flags/blocks.rs

```diff
@@ -136,7 +136,8 @@ impl Blocks {
 
     /// Prepends a [Block] of variant [INode](Block::INode) to `self`.
     fn prepend_inode(&mut self) {
-        self.0.insert(0, Block::INode)
+        self.0.insert(0, Block::INode);
+        self.0.insert(1, Block::INodeCount)
```

Comment:
As of now, I think we can have `inode-count` enabled only when they specify it explicitly in blocks. Let `--inode` only include the `inode` block.

Reply:
But the number of hard links is only show when the inode option is selected they, are tied in together. I could seperate and create a `prepend_inode_count()` method to do so and call it after the `prepend_inode()` method, would that be alright?

Reply:
What I meant was we can leave the `--inode` flag as is and have `links`(`inode-count`) show up only when it is passed explicitly in `--blocks`. Something like `lsd --blocks name,size,links`.

Reply:
I had a doubt, `links` should only be shown with the long option or also as is when passed to blocks.

Reply:
Actually both, but that is part of a separate logic and you don't really have to handle that. All you have to do is to add a new block and the logic to parse the `links` block in from config and cli. Something like [this](https://github.com/meain/lsd/blob/e3649fede8f7fc64041b709c56a3438ddbf813f0/src/flags/blocks.rs#L174)